### PR TITLE
Simplify the conflict layout.

### PR DESCRIPTION
**Pull request by hotchemi**
Just a tiny little bit customize, make it simple and modest expression.

image

### CODE REVIEW

**Comment by hkurokawa** [comment]
Great! Thanks :ok_woman:

**Comment by konifar** [comment]
LGTM!

**Comment by hotchemi** [comment]
Sorry just added unrelated commit, cuz this is my last pr!

File: app/src/main/java/io/github/droidkaigi/confsched/fragment/MyScheduleSessionsTabFragment.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ protected SessionsAdapter createAdapter() {
 
     private class MyScheduleAdapter extends SessionsAdapter {
 
-        private Map<Long, Pair<Integer, Integer>> rangeMap = new HashMap<>();
+        private Map<Long, Pair<Integer, Integer>> rangeMap = new ArrayMap<>();
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/MyScheduleSessionsTabFragment.java]: ArrayMap is better for not huge size map.

File: app/src/main/java/io/github/droidkaigi/confsched/fragment/StackedPageListener.java [hunk 2]
```diff
@@ -4,5 +4,5 @@
  * @author KeishinYokomaku
  */
 public interface StackedPageListener {
-	void onTop();
+    void onTop();
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/fragment/StackedPageListener.java]: Just applied formatter.

**Comment by konifar** [comment]
Thanks!!

**Comment by konifar** [comment]
LGTM

**Pull request merged**